src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -445,6 +571,214 @@ private string ClearHexString(string hexString)
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+            return ScriptsToOpCode(scripts);
+        }
+
+        /// <summary>
+        /// Converts a Hex Scripts to OpCode
+        /// </summary>
+        /// <param name="hex">Scripts in Hex string</param>
+        /// <returns>Returns null when the string does not represent a scripts; otherwise
+        /// returns the string that represents the converted OpCode.
+        /// </returns>
+        public string? HexScriptsToOpCode(string hex)
+        {
+            List<byte> scripts;
+            try
+            {
+                scripts = hex.HexToBytes().ToList();
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+            try
+            {
+                _ = new Script(scripts.ToArray(), true);
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+            return ScriptsToOpCode(scripts);
+        }
+
+        private string ScriptsToOpCode(List<byte> scripts)
+        {
+            //Initialize all OpCodes
+            var OperandSizePrefixTable = new int[256];
+            var OperandSizeTable = new int[256];
+            foreach (FieldInfo field in typeof(OpCode).GetFields(BindingFlags.Public | BindingFlags.Static))
+            {
+                var attribute = field.GetCustomAttribute<OperandSizeAttribute>();
+                if (attribute == null) continue;
+                int index = (int)(OpCode)field.GetValue(null);
+                OperandSizePrefixTable[index] = attribute.SizePrefix;
+                OperandSizeTable[index] = attribute.Size;
+            }
+            //Initialize all InteropService
+            var dic = new Dictionary<uint, string>();
+            ApplicationEngine.Services.ToList().ForEach(p => dic.Add(p.Value.Hash, p.Value.Name));
+
+            //Analyzing Scripts
+            var result = new List<string>();
+            while (scripts.Count > 0)
+            {
+                var op = (OpCode)scripts[0];
+                var operandSizePrefix = OperandSizePrefixTable[scripts[0]];
+                var operandSize = OperandSizeTable[scripts[0]];
+                scripts.RemoveAt(0);
+
+                var onlyOpCode = true;
+                if (operandSize > 0)
+                {
+                    var operand = scripts.Take(operandSize).ToArray();
+                    if (op.ToString().StartsWith("PUSHINT"))
+                    {
+                        result.Add($"{op} {new BigInteger(operand)}");
+                    }
+                    else if (op == OpCode.SYSCALL)
+                    {
+                        result.Add($"{op} {dic[BitConverter.ToUInt32(operand)]}");
+                    }
+                    else
+                    {
+                        result.Add($"{op} {operand.ToHexString()}");
+                    }
```

Comment:
We may extend the list of available formatters, take a look at the https://github.com/nspcc-dev/neo-go/blob/198af7f3aef039995b8f8cacefefb3a807b41ff3/pkg/vm/vm.go#L184-L230